### PR TITLE
FEATURE: Break on whitespace on html preview

### PR DESCRIPTION
**Pull request by jonnitto**
In the HTML preview, the source is often all in one line, which is not very readable. This PR set the style `white-space: break-spaces` on the `pre` tag (only in the `HTML` tab)

Before:
before

After:
after

### CODE REVIEW

**Comment by jonnitto** [comment]
Should I also build the sources, @mficzel?

**Comment by mficzel** [comment]
Yes please add the new sources

File: Resources/Private/JavaScript/containers/styleguide/main/info-tabs/code/index.js [hunk 1]
```diff
@@ -6,11 +6,12 @@ import codeStyle from './codeStyle';
 export default class Code extends Component {
     static propTypes = {
         content: PropTypes.string,
-        language: PropTypes.string
+        language: PropTypes.string,
+        style: PropTypes.array
```
Comment (grebaldi) [inline Resources/Private/JavaScript/containers/styleguide/main/info-tabs/code/index.js:10]: Shouldn't this be `PropTypes.object`?

Comment (mficzel) [reply Resources/Private/JavaScript/containers/styleguide/main/info-tabs/code/index.js:10]: @jonnitto ☝️

Comment (jonnitto) [reply Resources/Private/JavaScript/containers/styleguide/main/info-tabs/code/index.js]: Done

**Review by grebaldi — APPROVED** [review]

**Pull request merged**